examples/repl/cmdline.rs

```diff
@@ -138,28 +138,27 @@ unsafe fn poke_spec(context: &Context, spec: *const libc::c_char) -> libc::c_int
             .sql
             .set_config(context, "import_spec", Some(as_str(real_spec)))
             .unwrap();
-        current_block = 7149356873433890176;
+        ok_to_continue = true;
```

Comment:
I think for the ok_to_continue pattern it's fine to initialize it as true and only set it to false if needed. Setting it manually to true is not really needed but still fine.